javacutil/src/main/java/org/checkerframework/javacutil/ElementUtils.java

```diff
@@ -55,10 +55,22 @@ private ElementUtils() {
      *
      * @param elem the enclosed element of a class
      * @return the innermost type element, or null if no type element encloses {@code elem}
+     * @deprecated use {@link #enclosingType}
      */
+    @Deprecated // use enclosingType
     public static @Nullable TypeElement enclosingClass(final Element elem) {
+        return enclosingType(elem);
+    }
+
+    /**
+     * Returns the innermost type element enclosing the given element.
+     *
+     * @param elem the enclosed element of a class
+     * @return the innermost type element, or null if no type element encloses {@code elem}
+     */
+    public static @Nullable TypeElement enclosingType(final Element elem) {
```

Comment:
```
class Outer {
    static class Inner {  }
}
```
If the element is `Inner` this method returns `Outer`, which is not the enclosing type of  `Inner`.  Inner has no enclosing type.  (See `DeclaredType#getEnclosingType`.) I would name this method `enclosingTypeElement`.